src/Components/test/testassets/TestServer/CorsStartup.cs

```diff
@@ -33,31 +42,21 @@ public void Configure(IApplicationBuilder app, IWebHostEnvironment env)
                 app.UseDeveloperExceptionPage();
             }
 
-            // It's not enough just to return "Access-Control-Allow-Origin: *", because
-            // browsers don't allow wildcards in conjunction with credentials. So we must
-            // specify explicitly which origin we want to allow.
-            app.UseCors(policy =>
-            {
-                policy.SetIsOriginAllowed(host => host.StartsWith("http://localhost:") || host.StartsWith("http://127.0.0.1:"))
-                    .AllowAnyHeader()
-                    .WithExposedHeaders("MyCustomHeader")
-                    .AllowAnyMethod()
-                    .AllowCredentials();
-            });
-
             // Mount the server-side Blazor app on /subdir
             app.Map("/subdir", app =>
             {
                 app.UseStaticFiles();
                 app.UseClientSideBlazorFiles<BasicTestApp.Startup>();
 
                 app.UseRouting();
+
+                app.UseCors();
```

Comment:
We have controllers that use CORS and use the `AllowAll` policy. The change picked up an incorrectly configured app.